demo/src/main/java/com/novoda/storagepathfinder/demo/FileSizeFinder.java

```diff
@@ -0,0 +1,66 @@
+package com.novoda.storagepathfinder.demo;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+
+final class FileSizeFinder {
+
+    private FileSizeFinder() {
+        // Uses static factory methods.
+    }
+
+    /**
+     * Counts the size of a directory recursively (sum of the length of all files).
```

Comment:
I'd say "Calculates" instead of "Counts"

Reply:
3ac57d5